amethyst_network/src/test.rs

```diff
@@ -94,7 +76,7 @@ mod test {
             }
         }
         cl_dispatch.dispatch(&mut world_cl.res);
-        sleep(Duration::from_millis(100));
+        sleep(Duration::from_millis(10000));
```

Comment:
10 secs? Why so long? It's pretty annoying to have a test last so long.

Reply:
Ah, it was for testing, should be changed to 100.